doc/examples/streamline_length.py

```diff
@@ -86,8 +86,8 @@ def simulated_bundles(no_streamlines=50, n_pts=100):
 
 """
 Often, streamlines are represented with more points than what is actually
-necessary for specific applications. Also, sometimes every streamline has
-different number of points which could be of a trouble for some algorithms.
+necessary for specific applications. Also, sometimes every streamline has a
+different number of points which could be a problem for some algorithms.
```

Comment:
```suggestion
different number of points, which could be a problem for some algorithms.
```
Always a comma before "which"